<ui version="4.0" >
 <class>OsmSaveDlg</class>
 <widget class="QDialog" name="OsmSaveDlg" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Save OSM</string>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="whatsThis" >
   <string/>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Where to save:</string>
       </property>
       <property name="indent" >
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="OSMFileEdit" />
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="browseOSMButton" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Features to save:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="leftMargin" >
      <number>15</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="chkPoints" >
         <property name="text" >
          <string>Points</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkLines" >
         <property name="text" >
          <string>Lines</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkPolygons" >
         <property name="text" >
          <string>Polygons</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="chkRelations" >
         <property name="text" >
          <string>Relations</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkTags" >
         <property name="text" >
          <string>Tags</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox" >
         <property name="maximumSize" >
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>OSMFileEdit</tabstop>
  <tabstop>browseOSMButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OsmSaveDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
